### PR TITLE
add wasm_runtime_get_cur_local_obj_ref API support and change wasm runtime APIs name.

### PR DESCRIPTION
**Pull request by LevelCA**

### CODE REVIEW

File: core/iwasm/common/gc/gc_common.c [hunk 1]
```diff
@@ -892,6 +892,15 @@ wasm_runtime_pop_local_object_refs(WASMExecEnv *exec_env, uint32 n)
     } while (--n > 0);
 }
 
+WASMLocalObjectRef *
+wasm_runtime_get_current_local_object_ref(WASMExecEnv *exec_env)
```
Comment (xujuntwt95329) [inline core/iwasm/common/gc/gc_common.c:896]: The `current` in function name seems unnecessary. How about rename it as `wasm_runtime_get_local_object_ref`?

Comment (xujuntwt95329) [reply core/iwasm/common/gc/gc_common.c:896]: And do you want to get non-first ref?
e.g.

``` C
wasm_runtime_get_local_object_ref(exec_env, 0);
wasm_runtime_get_local_object_ref(exec_env, 1);
wasm_runtime_get_local_object_ref(exec_env, 2);
...
```

Comment (LevelCA) [reply core/iwasm/common/gc/gc_common.c:896]: `wasm_runtime_get_local_object_ref`  the name is ok. I just need to get the current ref, as I continue to push into ref, when I need to pop, I can continue to move the pointer until it is consistent with the current pointer I obtained before, which means that I have popped all the obj that I pushed before.

Comment (LevelCA) [reply core/iwasm/common/gc/gc_common.c:896]: @xujuntwt95329

Comment (wenyongh) [reply core/iwasm/common/gc/gc_common.c:896]: How about `wasm_runtime_cur_local_object_ref`?

Comment (LevelCA) [reply core/iwasm/common/gc/gc_common.c:896]: `wasm_runtime_get_cur_local_object_ref ` is better, what do you think?

Comment (wenyongh) [reply core/iwasm/common/gc/gc_common.c]: It is OK to me. BTW, should we shorten the function names, like:
```C
wasm_runtime_push_local_obj_ref
wasm_runtime_pop_local_obj_ref
wasm_runtime_pop_local_obj_refs
wasm_runtime_get_cur_local_obj_ref
```
@xujuntwt95329

Comment (xujuntwt95329) [reply core/iwasm/common/gc/gc_common.c]: > It is OK to me. BTW, should we shorten the function names, like:
> 
> ```c
> wasm_runtime_push_local_obj_ref
> wasm_runtime_pop_local_obj_ref
> wasm_runtime_pop_local_obj_refs
> wasm_runtime_get_cur_local_obj_ref
> ```
> 
> @xujuntwt95329

These names seems better

Comment (LevelCA) [reply core/iwasm/common/gc/gc_common.c]: OK, thanks, done

Comment (wenyongh) [reply core/iwasm/common/gc/gc_common.c]: @LevelCA should also change wasm_runtime_push_local_object_ref/wasm_runtime_pop_local_object_ref/wasm_runtime_pop_local_object_refs to wasm_runtime_push_local_obj_ref/wasm_runtime_pop_local_obj_ref/wasm_runtime_pop_local_obj_refs, and the related codes in core/iwasm?

Comment (LevelCA) [reply core/iwasm/common/gc/gc_common.c]: @wenyongh e. it's my fault.  All have been modified. thks

File: core/iwasm/include/gc_export.h [hunk 2]
```diff
@@ -910,7 +910,18 @@ wasm_runtime_pop_local_object_ref(wasm_exec_env_t exec_env);
  * @param n number to pop
  */
 WASM_RUNTIME_API_EXTERN void
-wasm_runtime_pop_local_object_refs(wasm_exec_env_t exec_env, uint32_t n);
+wasm_runtime_pop_local_obj_refs(wasm_exec_env_t exec_env, uint32_t n);
+
+/**
+ * get current local object ref from stack
```
Comment (wenyongh) [inline core/iwasm/include/gc_export.h:916]: Could you change to `Get` as it is upper case in other places?

**Review by wenyongh — COMMENTED** [review]
LGTM with minor comment

File: core/iwasm/include/gc_export.h (hunk 2 above)
Comment (LevelCA) [reply core/iwasm/include/gc_export.h]: aha, OK,  done!

**Review by xujuntwt95329 — COMMENTED** [review]
LGTM

**Pull request merged**